_assets/patches/geth/README.md

```diff
@@ -27,18 +30,21 @@ When a new stable release of `go-ethereum` comes out, we need to upgrade our for
 #### I. In our fork at /status-im/go-ethereum.
 
 1. Remove the local `develop` branch.
+
 ```bash
 git branch -D develop
 ```
 
-2. Pull upstream release branch into `develop` branch.
+1. Pull upstream release branch into `develop` branch.
```

Comment:
Why did you change it?

Reply:
Because the best practice in markdown files is to always use `1.`. The markdown renderer will take care of putting the right numbers, and that way you can insert new items into a list and not have to care about manually renumbering paragraphs.